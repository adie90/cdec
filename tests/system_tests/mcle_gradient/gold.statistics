-lm_nodes 4
-lm_edges 8
-lm_paths 5
-lm_trans A
-lm_trans 0.652536
constr_nodes 5
constr_edges 6
constr_paths 2
-lm_nodes 4
-lm_edges 8
-lm_paths 5
-lm_trans A
-lm_trans 0.652536
constr_nodes 5
constr_edges 5
constr_paths 1
-lm_nodes 7
-lm_edges 17
-lm_paths 15
-lm_trans C A
-lm_trans 0.74624
constr_nodes 8
constr_edges 13
constr_paths 6
