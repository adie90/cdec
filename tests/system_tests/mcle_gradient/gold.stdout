0	**OBJ**=3.05441;RewriteXY=-0.123601;SomeFeat=0.431589;F1=0.139951;F2=0.05909;F3=-0.116843;F4=0.106688;F5=0.0811144;F6=-0.257667;F7=-0.0224891;F8=0.118842;F9=-0.545223;F10=0.473137;F11=0.0176334;F12=-0.210479;F13=0.0162935
